sound/soc/sof/intel/hda-loader.c

```diff
@@ -145,8 +145,11 @@ static int cl_dsp_init(struct snd_sof_dev *sdev, int stream_tag)
 				       chip->ipc_ack_mask,
 				       chip->ipc_ack_mask);
 
-	/* step 5: power down corex */
-	ret = hda_dsp_core_power_down(sdev, chip->host_managed_cores_mask & ~(BIT(0)));
+	/* step 5: power down corex except init corex */
```

Comment:
@kv2019i the X in the corex is just a substitue for the core IDs. FOr example on CNL, corex would be cores 2-4.

Reply:
@ranj063 That still leaves we wondering what is X, and whether this adds value to understanding the code. It seems this is not as if I "open code" the new comment, it would be for CNL "step 5: power down cores 2-4 except init cores 2-4". So it doesn't make sense unless you just read "corex" as some set of cores. Oh well, I can live this, but it seems like something that might have made sense at start, but now with more and more people modifying the code, is no longer helpful.

I'd propose following "open-coding": "step 5: power down cores that are no longer needed"

Reply:
yes, sounds good. We should remove all ambiguities. In fact this step is a bit controversial already. For example on CNL+ platforms, you cannot really power off cores 2-4 ie writing 0 to the SPA bits for those cores has no effect.

Reply:
maybe the host_managed_cores handles this. something to check

Reply:
FWIW "corex" confused me too. So, I understand another revision will be posted with those removed, also please remember to update the commit title accordingly

Reply:
@kv2019i @ranj063 @lyakh Updated.